packages/mobile/src/config.ts

```diff
@@ -156,6 +157,8 @@ export const CASH_IN_FAILURE_DEEPLINK = 'celo://wallet/cash-in-failure'
 
 export const APP_STORE_ID = Config.APP_STORE_ID
 export const DYNAMIC_DOWNLOAD_LINK = Config.DYNAMIC_DOWNLOAD_LINK
+export const CROWDIN_DISTRIBUTION_HASH = 'e-f9f6869461793b9d1a353b2v7c'
+export const OTA_TRANSLATIONS_FILEPATH = `file://${DocumentDirectoryPath}/translations`
```

Comment:
Should we use the Cache directory instead?

> Data that can be downloaded again or regenerated should be stored in the <Application_Home>/Library/Caches directory. Examples of files you should put in the Caches directory include database cache files and downloadable content, such as that used by magazine, newspaper, and map applications.

Source: https://developer.apple.com/icloud/documentation/data-storage/

Reply:
updated!